chart/templates/nginx.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-config
+  namespace: {{ .Release.Namespace }}
+data:
+  nginx.conf: |
+    events {
+        worker_connections  1024;
+    }
+
+    stream {
+        map $ssl_preread_server_name $selected_upstream {
+            hostnames;  
+            *.admin.uds.dev backend_admin;
+            keycloak.uds.dev backend_keycloak;
```

Comment:
Not really because (as you'll see with my UDS Core keycloak work, we only will be doing single-host vs wildcard for passthrough and keycloak will be the only one. Cant change the name as it will break keyloak if it's different than expected.  So--passthrough is the name of the _gateway_ and _keycloak_ is the name of first (currently only) passhthrough host we have in our test bed.

Reply:
Yeah sorry my comment wasn't clear - meant specifically the `backend_keycloak` name, with haproxy before the equivalent (`bk_http_passthrough`) was passthrough. Didn't mean to suggest the hostname should change.

NBD either way, I think this is fine if you want to keep.

Reply:
ah yes thats fair I can change that